runtime/src/main/java/org/corfudb/runtime/clients/LogUnitHandler.java

```diff
@@ -16,12 +13,10 @@
 import org.corfudb.protocols.wireprotocol.ReadResponse;
 import org.corfudb.protocols.wireprotocol.TailsResponse;
 import org.corfudb.runtime.exceptions.DataCorruptionException;
-import org.corfudb.runtime.exceptions.DataOutrankedException;
 import org.corfudb.runtime.exceptions.OutOfSpaceException;
 import org.corfudb.runtime.exceptions.OverwriteCause;
 import org.corfudb.runtime.exceptions.OverwriteException;
 import org.corfudb.runtime.exceptions.TrimmedException;
-import org.corfudb.runtime.exceptions.ValueAdoptedException;
 
```

Comment:
Looks like we can also delete `ValueAdoptedException` file?

Reply:
Or it could be handled in LogUnit RPC PR

Reply:
I removed it.

Reply:
Also it seems like you can remove `OutOfSpaceException` too its not being used, so it shouldn't be in the protobuf definitions.

Reply:
It hasn't been included in the Protobuf definitions.